src/lexedata/importer/interleaved.py

```diff
@@ -76,12 +76,28 @@
             if len(cogsets) == 1 or len(cogsets) == len(forms):
                 True
             else:
-                print(
+                logger.warning(
                     "{:}: Forms ({:}) did not match cognates ({:})".format(
                         entry.coordinate, ", ".join(forms), ", ".join(cogsets)
                     ),
                     file=sys.stderr,
                 )
+            for form, cogset in zip(forms, cogsets + [None]):
+                w.writerow([language_name, concepts[c], form, None, cogset])
+
+
+if __name__ == "__main__":
+    import argparse
```

Comment:
(This is one of the few scripts where I won't suggest using `cli.parser` instead, because this script is metadata-free.)

Reply:
Yes, I saw the cli.parser script, but I was still waiting for the merge of the pull request